tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-inference-viewer.html

```diff
@@ -0,0 +1,233 @@
+<!--
+@license
+Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<link rel="import" href="../iron-icons/iron-icons.html">
+<link rel="import" href="../polymer/polymer.html">
+
+<!-- Displays the results of inferences in the what-if tool side panel. -->
+<dom-module id="tf-inference-viewer">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+      .runs-holder {
+        padding: 0;
+        display: flex;
+        flex-direction: column-reverse;
+        overflow-y: auto;
+        max-height: 200px;
+      }
+      .row {
+        display: flex;
+      }
+      .header {
+        color: grey;
+        font-weight: 500;
+        font-size: 12px;
+        padding: 5px 0;
+      }
+      .dark-row {
+        background: #cbf0f8
+      }
+      .stripe-row {
+        background: #e4f7fb;
+      }
+      .white-row {
+        background: white;
+      }
+      .marker {
+        flex: 0.5;
+        margin-right: 5px;
+      }
+      .marker.mark {
+        background: #007B83;
+        padding-left: 1px;
+      }
+      .run {
+        flex: 10;
+      }
+      .label {
+        flex: 20;
+        display: flex;
+        overflow: hidden;
+      }
+      .no-label {
+        display: none;
+      }
+      .score {
+        flex: 10;
+      }
+      .arrow {
+        flex: 2;
+      }
+      .delta {
+        flex: 2;
```

Comment:
super nit: this property is overridden by next line

Reply:
done